src/Appwrite/Platform/Modules/Locale/Http/Locale/ListCountriesPhones.php

```
@@ -0,0 +1,66 @@
+            ->inject('locale')
+            ->callback($this->action(...));
+    }
+
+    public function action(UtopiaResponse $response, Locale $locale): void
+    {
+        $list = Config::getParam('locale-phones'); /* @var $list array */
+        $output = [];
+
+        \asort($list);
+
+        foreach ($list as $code => $name) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$name'. (undefined)

(UnusedLocalVariable)